docs/pages/installation/kinD.md

```diff
@@ -153,9 +156,9 @@ Run the following:
 
  <pre class="codeblock-pre"><div class="codeblock">
  <div class="clipboardjs">
- $ git clone https://github.com/layer5io/meshery.git; cd meshery
- $ kubectl create namespace meshery
- $ helm install meshery --namespace meshery install/kubernetes/helm/meshery
+$ kubectl create ns meshery

```

Comment:
@Aisuko?

Reply:
Hi, guys. In fact, we can use `helm install` to create the namespace.

For example, the command below can work well. But if we'd like create though `kubectl`, it's ok.

```
helm install meshery . -n meshery --create-namespace
```

Reply:
It would be nice to reduce the number of commands that users have to execute. If the `helm` command can create the namespace, let's consolidate commands.

Reply:
Though people not suggest to create a name space using Helm v3, we can actually do it... Been exploring Helm charts due to my clg project 😂 but came to know a lot of things. Let me try such method and get back within today